x-pack/plugins/enterprise_search/public/applications/applications/components/engine/engine_indices.tsx

```diff
@@ -174,24 +178,31 @@ export const EngineIndices: React.FC = () => {
   ];
   return (
     <>
-      {hasUnknownIndices && (
+      {(hasAllUnreachableIndices || hasUnknownIndices) && (
         <>
           <EuiCallOut
             color="warning"
             iconType="warning"
-            title={i18n.translate(
-              'xpack.enterpriseSearch.content.engine.indices.unknownIndicesCallout.title',
-              { defaultMessage: 'Some of your indices are unavailable.' }
-            )}
+            title={
+              <FormattedMessage
+                id="xpack.enterpriseSearch.content.engine.indices.unknownIndicesCallout.title"
+                defaultMessage="{title} of your indices are unavailable."
+                values={{
+                  title: hasAllUnreachableIndices ? 'All' : 'Some',
+                }}
```

Comment:
this approach (lines 199-203 included) would produce missing translations for the words `"All"` and `"Some"`. you'll either have to wrap these in their own `FormattedMessage`/`i18n.translate` _or_ create entire separate FormattedMessage/i18n.translate calls and move the `hasAllUnreachableIndices ?` ternary up a bit.

my preference is for the latter because then translators will have the full grammatical context of the sentence when translating

Reply:
Add in 8c56be8e34e27add99377f60f6bfed2eaf654abf